experiment/run_experiment.py

```diff
@@ -473,14 +473,18 @@ def main():
         'more benchmarks.')
 
     all_benchmarks = benchmark_utils.get_all_benchmarks()
-    all_fuzzers = fuzzer_utils.get_fuzzer_names()
-
+    code_coverage_benchmarks = [
```

Comment:
This should be a helper function in benchmark_utils, just like one that just adds these two - get_oss_fuzz_coverage_benchmarks(),get_standard_coverage_benchmarks().